p5/sketch/Skia2DRenderer/renderer2d.py

```diff
@@ -704,6 +707,42 @@ def end_shape(
                             self.path.lineTo(v[0], v[1])
                 self._do_fill_stroke_close(close_shape)
 
-    # Fonts functions
-
     # Images functions
+    def create_image(self, width, height):
+        return SkiaPImage(width, height)
+
+    def image(self, pimage, x, y, w=None, h=None):
+        if self.style.image_mode == constants.CORNER:
+            if w and h:
+                pimage.size = (w, h)
+        elif self.style.image_mode == constants.CORNERS:
+            pimage.size = (w - x, h - y)
+        elif self.style.image_mode == constants.CORNERS:
```

Comment:
This condition and the one above look like they are the same.

Reply:
Oops, that should be `CENTER`. Thanks!